README.md

```diff
@@ -1,70 +1,125 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Rent-A-Green
```

Comment:
- [ ] In your readme, kindly consider adding a screenshot.  Your project is beautiful.  Let's show it up! 💪